### ISSUE TITLE
Datagrid column toggler doesn't announce updates

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3578]
## Describe the bug

Screen reader users do not hear any status updates after the checkboxes in the clr-checkbox-wrapper components are activated in side of the column toggler.

## How to reproduce

https://stackblitz.com/github/clane/clarityTest2/tree/master/clarity-test-app

- Go to "Data Grid - Hide/Show" example
- Using a screen reader, activate the clr-dg-column-toggle component
- Check and uncheck any of the checkboxes in the "switch-content" 
- Observe that you do not hear any information about how the grid contents change as you interact with these checkboxes.

## Expected behavior

Implement an ARIA live region to announce grid updates as status updates.

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by Jinnie** [comment]
I believe it was fixed as part of #3537 
The problem was that checkbox elements were recreated by Angular on toggle, because the data model changed. This was leading to loss of focus from the checkboxes, which prevented the readers from reading status updates. 
The fix was to add a trackBy function and preserve the rendered elements.
Now it announces selection like this - name of column, action taken, "checkbox":
<img width="694" alt="Screen Shot 2019-07-03 at 1 44 08 PM" src="https://user-images.githubusercontent.com/1798828/60585672-ec815700-9d98-11e9-9f2d-befa9996e9f3.png">

Please reopen if it does not satisfy the issue expectations.

**Comment by chlane** [comment]
this defect wasn't about simply having screen readers announce the checked state but to announce as status message saying that the columns have changed
